### PR TITLE
Creates new federal penitentiary in Rock Springs

### PR DESCRIPTION
**Pull request by SigholtStarsong**
## About The Pull Request

This PR changes the RockSprings and RockSprings-upper Z-levels to include an all-new prison dungeon. This dungeon has a mix of raiders and renegade mobs with unique names, all added to the "Powder" faction. It also makes better use of a considerable portion of the map on RockSprings.

## Why It's Good For The Game

This creates a new dungeon that supports AI guards for a Powder Ganger faction.

## Pre-Merge Checklist

- [ x] You tested this on a local server.
- [ x] This code did not runtime during testing.
- [ x] You documented all of your changes.

## Changelog

:cl:
add: New two-level prison
add: Secret megafauna dungeon in Rock Springs
add: New secret chainsaw spawn
del: Removed overpass
del: removed raider shack where prison is going
tweak: Moved another raider shack north of the river, keeping it in place.
/:cl:

### CODE REVIEW

**Comment by SigholtStarsong** [comment]
Eugh, damn you mapdiffbot.

**Review by PolymorphBlue — APPROVED** [review]
Map changes, I'm trusting you here.

File: _maps/map_files/Pahrump-Sunset/RockSprings.dmm [hunk 1]
```diff
@@ -14599,6 +15490,13 @@
 	icon_state = "plating"
 	},
 /area/f13/wasteland)
+"sZx" = (
+/obj/structure/ladder/unbreakable{
+	id = PrisonService;
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings.dmm:15495]:
```suggestion
	id = "PrisonService";
```

File: _maps/map_files/Pahrump-Sunset/RockSprings.dmm [hunk 2]
```diff
@@ -695,6 +749,18 @@
 	icon_state = "verticalinnermain0"
 	},
 /area/f13/wasteland)
+"aYb" = (
+/obj/structure/bed/wooden,
+/obj/item/nullrod/tribal_knife{
+	desc = 1"They say fear is the true mind killer, but stabbing them in the head works too. Honour compels you to not sheathe it once drawn.";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings.dmm:755]:
```suggestion
	desc = "They say fear is the true mind killer, but stabbing them in the head works too. Honour compels you to not sheathe it once drawn.";
```

**Review by out-of-phaze — CHANGES_REQUESTED** [review]
Changes necessary to fix the linter, don't worry, I've got it!

File: _maps/map_files/Pahrump-Sunset/RockSprings.dmm [hunk 3]
```diff
@@ -13145,17 +13976,23 @@
...
 "rnh" = (
 /turf/open/indestructible/ground/outside/dirt/harsh{
 	dir = 5;
 	icon_state = "dirt"
 	},
 /area/f13/wasteland)
+"rnt" = (
+/mob/living/simple_animal/hostile/vault/security{
+	desc = "Imagine being a criminal and dressing as a guard. What a joke.";
+	faction = list("powder";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings.dmm:13991]:
```suggestion
	faction = list("powder");
```

File: _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm [hunk 4]
```diff
@@ -630,6 +980,17 @@
 /obj/effect/decal/cleanable/dirt,
 /turf/open/floor/wood/wood_wide,
 /area/f13/wasteland)
+"jx" = (
+/obj/structure/chair/wood/dining{
+	dir = 8
+	},
+/mob/living/simple_animal/hostile/vault/security{
+	faction = list("powder";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm:988]:
```suggestion
	faction = list("powder");
```

File: _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm [hunk 5]
```diff
@@ -770,6 +1239,14 @@
 	icon_state = "yellowdirtyfull"
 	},
 /area/f13/wasteland)
+"lA" = (
+/mob/living/simple_animal/hostile/vault/security{
+	faction = list("powder";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm:1244]:
```suggestion
	faction = list("powder");
```

File: _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm [hunk 6]
```diff
@@ -1705,10 +2756,44 @@
...
 /area/f13/wasteland)
+"zc" = (
+/obj/effect/turf_decal/trimline/neutral/line{
+	dir = 6
+	},
+/turf/open/indestructible/ground/outside/sidewalk,
+/area/f13/building)
+"zj" = (
+/mob/living/simple_animal/hostile/vault/security{
+	faction = list("powder";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm:2767]:
```suggestion
	faction = list("powder");
```

File: _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm [hunk 7]
```diff
@@ -1705,10 +2756,44 @@
...
+/obj/structure/chair/comfy/beige,
+/turf/open/floor/wood/wood_fancy/wood_fancy_dark,
+/area/f13/building)
 "zF" = (
 /obj/effect/decal/cleanable/dirt,
 /turf/open/floor/wood/wood_mosaic,
 /area/f13/wasteland)
+"zH" = (
+/mob/living/simple_animal/hostile/vault/security{
+	faction = list("powder";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm:2787]:
```suggestion
	faction = list("powder");
```

File: _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm [hunk 8]
```diff
@@ -1788,6 +2903,34 @@
...
+/obj/structure/junk/locker,
+/turf/open/floor/plasteel/f13/vault_floor/misc/rarewhite{
+	icon_state = "showroomfloor";
+	name = "tile"
+	},
+/area/f13/building)
+"Bq" = (
+/obj/effect/decal/cleanable/blood/tracks,
+/mob/living/simple_animal/hostile/vault/security{
+	faction = list("powder";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm:2928]:
```suggestion
	faction = list("powder");
```

File: _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm [hunk 9]
```diff
@@ -3053,6 +4816,18 @@
 	},
 /turf/open/floor/wood/wood_common/wood_common_light,
 /area/f13/wasteland)
+"Zs" = (
+/obj/machinery/shower{
+	dir = 4
+	},
+/obj/machinery/pool/drain,
+/mob/living/simple_animal/hostile/vault/security{
+	faction = list("powder";
```
Comment (out-of-phaze) [inline _maps/map_files/Pahrump-Sunset/RockSprings-Upper.dmm:4825]:
```suggestion
	faction = list("powder");
```

**Review by out-of-phaze — CHANGES_REQUESTED** [review]
More `)`s.

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**